flow/include/flow/genericactors.actor.h

```diff
@@ -115,6 +116,21 @@ std::vector<T> parseStringToVector(std::string str, char delim) {
 	return result;
 }
 
+template <class T>
+std::unordered_set<T> parseStringToUnorderedSet(std::string str, char delim) {
+	std::unordered_set<T> result;
+	std::stringstream stream(str);
+	std::string token;
+	while (stream.good()) {
+		getline(stream, token, delim);
+		std::istringstream tokenStream(token);
+		T item;
+		tokenStream >> item;
```

Comment:
Thanks for adding the utility method; one thing worth considering is `white space` handling; current approach is to let the clients sanitize tokens with any leading and/or trailing white-spaces, an alternate could be to cleanup if client wants it to be done (function input parameter). 

Both approaches are acceptable, I think we should document the one followed here for more clarity.

Reply:
The istringstream removes leading/trialling whitespace when adding the element to the set, i can document this though for the method